media/js/base/global.js

```diff
@@ -114,10 +114,11 @@ function isFirefoxUpToDate(latest) {
     return (latestFirefoxVersion <= fx_version);
 }
 
-// Used on the plugincheck page to also support SeaMonkey
-function isSeaMonkey(userAgent) {
+// Used on the plugincheck page to also support all browsers based on Gecko.
+// Not including Camino as it has reached EOL [http://caminobrowser.org/]
+function isLikeFirefox(userAgent) {
     var ua = userAgent || navigator.userAgent;
-    return /SeaMonkey/i.test(ua);
+    return (/Iceweasel/i).test(ua) || (/SeaMonkey/i).test(ua);
```

Comment:
Ok so, apparently IceWeasel is now IceCat and although it does have a IceCat string in the userAgent it also uses the like Firefox string. see here:
http://useragentstring.com/pages/Browserlist/

So, we can either check specifically for IceCat and avoid including Camino or just use 'like Firefox' which will then include IceCat and Camino.

I am thinking let's check for IceCat specifically. Thoughts?